buildSrc/src/main/kotlin/dev/engine_room/gradle/transitive/TransitiveSourceSetsExtension.kt

```diff
@@ -25,7 +26,7 @@ open class TransitiveSourceSetsExtension(private val project: Project) {
 
     fun createCompileConfigurations() {
         val configs = transitives.mapValues { (sourceSet, _) ->
-            project.configurations.create("for${sourceSet.name.capitalize()}") {
+            project.configurations.create("for${sourceSet.name.replaceFirstChar { it.uppercase() }}") {
```

Comment:
Why was this change made?

Reply:
capitalize is deprecated, it shouldn't be used anymore